Tools/cases_generator/tier2_generator.py

```diff
@@ -38,10 +38,10 @@ def declare_variable(
 ) -> None:
     if var.name in variables:
         return
-    type = var.type if var.type else "PyObject *"
+    type = var.type if var.type else "_PyStackRef "
```

Comment:
I need the trailing space because the current codegen is 
``{type}{varname}```
which generates for example
```PyObject *hi```
but if I use _PyStackRef, it generates
```_PyStackRefhi```
causing a compiler error.

Reply:
What's wrong with the changing the tests a bit?
There should be almost no `PyObject *` in the generated code now, so there should be much of a change to the tests needed.

If you really don't want to change the tests at all, how about
````
space = "" if type.endswith("*") else " "
{type}{space}{varname}
```

Reply:
Ok done.